python/turbodbc/cursor.py

```diff
@@ -303,7 +303,7 @@ def _numpy_batch_generator(self):
             result_batch = _make_masked_arrays(numpy_result_set.fetch_next_batch())
             is_empty_batch = (len(result_batch[0]) == 0)
             if is_empty_batch and not first_run:
-                raise StopIteration # Let us return a typed result set at least once
+                return # Let us return a typed result set at least once
```

Comment:
Since you write "conform to PEP-479" in the change log, I would have expected a new test that would have broken with the previous code

Reply:
The code already failed on Python 3.7 in a lot of tests thus I have not added a new one.

Reply:
Good to know, that's sufficient, then.